internal/util/copy.go

```diff
@@ -83,6 +99,45 @@ func CopyIndexFiles(logger *log.CmdLogger, src, dest string) error {
 	return nil
 }
 
+// skipFolder checks if the folder will need to be skipped during indexes copy.
+func skipFolder(logger *log.CmdLogger, indexSrcDir, src string) (bool, error) {
+	path, _ := filepath.Rel(indexSrcDir, src)
+
+	matchContainsPublicIndexFolder, _ := filepath.Match("index*", path)
+	matchContainsPrivateDataCollectionFolder, _ := filepath.Match("collections*", path)
```

Comment:
Similarly `collections/*` although that's what you have for `matchPrivateDataCollectionFolder`. I don't think you need both unless I've misunderstood.

Reply:
The first folder after couchdb can be only collections or indexes.